### ISSUE TITLE
anaconda-build *requires* an underlying Git repository?

### ISSUE THREAD

**Issue by mcg1969** [Anaconda-Server/anaconda-build#187]
I have a `conda build` recipe that includes nothing but `meta.yaml` and `.binstar.yml` files. If I place them in an otherwise bare directory and run `anaconda-build submit .`, I see this:
```
Archiving build directory for upload ...
fatal: Not a git repository (or any of the parent directories): .git
Created archive (0 files); Uploading to binstar
 uploaded 2 of 2Kb: 100.00% ETA: 0.0 minutes
```
Not surprisingly, the builds fail because it can't find `meta.yaml`.

However, if I init a git repo, add the files to it, and commit, all is well:
```
Archiving build directory for upload ...
Created archive (2 files); Uploading to binstar
 uploaded 3 of 3Kb: 100.00% ETA: 0.0 minutes
```
Should the existence of a git repo be a requirement for anaconda-build submissions? I'd rather this not be the case. Indeed, I'd like to make sure the builds are *working* before I commit changes.

**Comment by mcg1969** [comment]
Looks like a duplicate of #174

**Comment by srossross** [comment]
@mcg1969 I think the message `fatal: Not a git repository (or any of the parent directories): .git` is coming from a command that is checking to see if this directory is a git repo. I think you should be able to ignore this. (I think this may be because it is checking your `.gitignore` file?)

This should still work with our without git, even with the error message.

In the future, I would like to add a `.anaconda-build-ignore` file to test what files to upload.

**Comment by mcg1969** [comment]
Agreed, the error message itself is fine with me, as long as the files actually get uploaded.
